acceptance/lib/helper.rb

```diff
@@ -30,7 +30,8 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "ba39f2f2fd1b4def9a7f366d469acb1c5f6db824", :string)
+                         "5cb5a63982c01f3ed0182c25b61213d208573d8d",
```

Comment:
Also, I think we want to update the value for `PUPPET_VERSION` above from`1.5.2.343.gba39f2f` to `1.5.2-411-g5cb5a63`.  TBH, though, I'm not sure why we have to do that part.  I had been thinking at one point that just leaving `PUPPET_VERSION` set to "" was fine - and that the right package to install could be gleaned just from the `PUPPET_BUILD_VERSION`.